app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/RejectionReason.tsx

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useMemo } from "react";
+import Alert from "react-bootstrap/Alert";
+import { LIFECYCLE_REVIEW_FLOWS } from "../../lib/constants";
+import { humanDate } from "../../lib/dateUtils";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import {
+  NimbusChangeLogOldStatus,
+  NimbusChangeLogOldStatusNext,
+} from "../../types/globalTypes";
+
+export const RejectionReason = ({
+  rejectionEvent,
+}: {
+  rejectionEvent: getExperiment_experimentBySlug["rejection"];
+}) => {
+  const { message, changedOn, changedBy, oldStatus, oldStatusNext } =
+    rejectionEvent!;
+
+  const rejectionActionDescription = useMemo(() => {
+    if (oldStatus === NimbusChangeLogOldStatus.LIVE) {
+      if (oldStatusNext === NimbusChangeLogOldStatusNext.LIVE) {
+        return LIFECYCLE_REVIEW_FLOWS.PAUSE.description;
+      }
+      return LIFECYCLE_REVIEW_FLOWS.END.description;
+    }
+    if (oldStatus === NimbusChangeLogOldStatus.DRAFT) {
+      return LIFECYCLE_REVIEW_FLOWS.LAUNCH.description;
+    }
+  }, [oldStatus, oldStatusNext]);
+
+  return (
+    <Alert variant="warning" data-testid="rejection-notice">
+      <div className="text-body">
+        <p className="mb-2">
+          The request to {rejectionActionDescription} was{" "}
+          <strong>Rejected</strong> due to:
```

Comment:
Oh @lmorchard I see what you were saying in planning, #5927 covers the other side of this, "You are rejecting..." instead of "The request to". Could easily be done in this PR or in that issue separately.

Reply:
Added a fix for this https://github.com/mozilla/experimenter/pull/5852/files#diff-c86b7877156b25769ecb6879ff40aef78f66ac82fa8ff115b6e679b6870a92b9R54-R57